librz/core/cmd_descs/cmd_history.yaml

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+# SPDX-License-Identifier: LGPL-3.0-only
+---
+name: cmd_history
+commands:
+  - name: "H"
+    cname: list_or_exec_history
```

Comment:
i prefer `history_list_or_exec`, `history_clear` and `history_save`. we usually put the "topic" as first word